dev/examples/data.js

```diff
@@ -1721,6 +1726,51 @@ const DATA = [
             showVGrid: true,
             showHGrid: true
         })
+    },
+    {
+        LABEL_TRUNCATION: (id) => ({
+            bindTo: id,
+            bindLegendTo: null,
+            axis: {
+                x: {
+                    show: true,
+                    label: largeLabel,
+                    onClick: loadXAndYAxisLabelPopup,
```

Comment:
are we wanting to expose the onClick event allowing consumers to override the behavior?  If so, why would we want to do that?  I'm assuming it was modeled after the setup for Gantt but we need to decide if the behavior should be the same.

If kept, this property should probably be `labelOnClick`.  The current property name indicates that the `onClick` is for the axis when it is only for the label.  The documentation for Axes would also need to be updated to reflect the new optional property.
https://github.com/cerner/carbon-graphs/blob/master/docs/helpers/Axes.md